test-suite/src/lib.rs

```diff
@@ -5,6 +5,7 @@ pub mod alter;
 pub mod arithmetic;
 pub mod ast_builder;
 pub mod basic;
+pub mod bit_shift;
```

Comment:
Not only adding a bit_shift module, we also need to use it.
you can see the code below, there's `glue!` macro and you can consider to use that in `generate_store_tests`